tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "compilerOptions": {
+    "experimentalDecorators": true,
+    "target": "es2017",
+    "allowJs": true,
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "noFallthroughCasesInSwitch": true,
+    "noUnusedLocals": true,
+    "noImplicitReturns": true,
+    // change to true when all files are switched over to ts
```

Comment:
Which files weren't converted?

Reply:
Comment was removed.  Left in by mistake.